third_party/mozdebug/Add_rust-lldb_debugger.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/mozdebug/mozdebug.py b/mozdebug/mozdebug.py
```

Comment:
Please place this in a `patches` directory and prefix it by the order that the patch should go in, so that we know what order to apply them when upgrading. So this would move to:

`third_party/mozdebug/patches/01-Add_rust-lldb_debugger.patch`

Thanks!